### PR TITLE
docs(en): add info about dynamic config

### PR DESCRIPTION
**Pull request by danielroe**
Matches https://github.com/nuxt/nuxt.js/pull/8520

### CODE REVIEW

File: content/en/guides/configuration-glossary/configuration-router.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ This can be useful if you need to serve Nuxt as a different context root, from w
 
 If you want to have a redirect to `router.base`, you can do so [using a Hook, see _Redirect to router.base when not on root_](/docs/2.x/configuration-glossary/configuration-hooks#redirect-to-routerbase-when-not-on-root).
 
+In Nuxt 2.15+, this property can be set using environment variables to change the configuration of an app that has already been built.
```
Comment (Atinux) [inline content/en/guides/configuration-glossary/configuration-router.md:22]: Can we specify the environnement variable name?

Comment (danielroe) [reply content/en/guides/configuration-glossary/configuration-router.md:22]: Maybe an example would be helpful? Something like:
```js
export default {
  router: {
    base: process.env.MY_BASE_URL || '/',
  }
}
```

File: content/en/guides/configuration-glossary/configuration-build.md [hunk 2]
```diff
@@ -593,6 +593,8 @@ export default {
 
 Then, when launching `nuxt build`, upload the content of `.nuxt/dist/client` directory to your CDN and voilà!
 
+In Nuxt 2.15+, changing the value of this property at runtime will override the configuration of an app that has already been built.
```
Comment (debs-obrien) [inline content/en/guides/configuration-glossary/configuration-build.md:596]: should this be a warning/info or best left as is??

Comment (danielroe) [reply content/en/guides/configuration-glossary/configuration-build.md:596]: I'm happy either way. I didn't see any other `<alert>` through the docs so removed my initial info alert...

**Pull request merged**